CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Pending
 
 - Implemented SHA-256 CRH
+- Fixed Pedersen padding bug [\#76](https://github.com/arkworks-rs/crypto-primitives/issues/76)
```

Comment:
Small nit, but could you change the format to match the other entries? So moving the following entries to the `Features` section?

```suggestion
- [\#77](https://github.com/arkworks-rs/crypto-primitives/pull/77) Implement SHA-256 CRH.
- [\#76](https://github.com/arkworks-rs/crypto-primitives/pull/79) Fix Pedersen padding bug.
```